C++/Indexed Reference List.tmPreferences

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>
+        source.c constant.other.macro.c,
+        source.c support.type.c,
+        source.c variable.other.indexed.c,
+        source.c variable.other.member.indexed.c
```

Comment:
These two `.indexed.` scopes can be removed if we use the standard scope in the definition

Reply:
Why not just drop the suffices altogether? You already have `source.c`; no need to specify the language scope at the end, too.

```
source.c variable.other
```

If there's some particular sub-scope that is unintentionally indexed, you can throw in an exception

```
source.c variable.other - variable.other.oops
```

Or even add [an extra file](https://github.com/braver/TypeScriptSyntax/pull/4/files#diff-5f4da57f87ace1671e461f40c7bbae7f) with the banned ones (e.g. `source.c variable.other.oops`) and `showInIndexedReferenceList` set to `0`.